src/status_im/contexts/settings/wallet/saved_addresses/sheets/share_address/view.cljs

```diff
@@ -0,0 +1,107 @@
+                          :button-label      (i18n/label :t/display)}])]
+    (rf/dispatch [:show-bottom-sheet
+                  {:theme   :dark
+                   :shell?  true
+                   :content sheet-content}])))
+
+(defn view
+  []
+  (let [padding-top                                (:top (safe-area/get-insets))
+        {:keys [name address customization-color]} (rf/sub [:get-screen-params])
+        [wallet-type set-wallet-type]              (rn/use-state :legacy)
+        [selected-networks set-selected-networks]  (rn/use-state constants/default-network-names)
+        on-settings-press                          (rn/use-callback #(open-preferences
+                                                                      selected-networks
+                                                                      set-selected-networks)
+                                                                    [selected-networks])
+        on-legacy-press                            (rn/use-callback #(set-wallet-type :legacy))
+        on-multichain-press                        (rn/use-callback #(set-wallet-type :multichain))
+        preferred-networks                         (rf/sub [:wallet/preferred-chain-names-for-address
+                                                            address])
+        share-title                                (str name " " (i18n/label :t/address))
+        qr-url                                     (rn/use-memo #(utils/get-wallet-qr
+                                                                  {:wallet-type       wallet-type
+                                                                   :selected-networks selected-networks
+                                                                   :address           address})
+                                                                [wallet-type selected-networks address])
+        qr-media-server-uri                        (rn/use-memo
+                                                    #(image-server/get-qr-image-uri-for-any-url
+                                                      {:url         qr-url
+                                                       :port        (rf/sub [:mediaserver/port])
+                                                       :qr-size     qr-size
+                                                       :error-level :highest})
+                                                    [qr-url])]
+
+    (rn/use-mount #(set-selected-networks preferred-networks))
+
+    [quo/overlay {:type :shell}
+     [rn/view
+      {:flex        1
+       :padding-top padding-top}
+      [quo/page-nav
+       {:icon-name           :i/close
+        :on-press            navigate-back
+        :background          :blur
+        :accessibility-label :top-bar}]
+      [quo/page-top
+       {:title           (i18n/label :t/share-address)
+        :container-style {:margin-bottom 8}}]
+      [rn/view {:style {:padding-horizontal 20}}
```

Comment:
Maybe we could extract these styles into a style namespace? wdyt?

Reply:
for a one-line style, I believe it's not needed.